public/templates/search_bar.html

```diff
@@ -1,9 +1,2 @@
-<div class="search_bar">
-  <input
-    type="search"
-    name="search-box"
-    value="{{this.pageQuery}}"
-    placeholder="Type to search"
-  />
-  <button type="submit" onclick="searchWeb()">search</button>
+{{>bar this}}
```

Comment:
Is this file still useful?

Reply:
Yes, but now that I think about it carefully, I feel that maybe I can remove this file and maybe edit the index.html file instead. So thanks :heart: for taking the time to review this PR, I really appreciate it :+1:  and thanks for pointing out this as well, I will make the change right away :slightly_smiling_face: .